lockbox-ios/Presenter/ItemListPresenter.swift

```diff
@@ -208,13 +210,35 @@ class ItemListPresenter {
         self.itemListDisplayActionHandler.invoke(ItemListSortingAction.alphabetically)
         self.itemListDisplayActionHandler.invoke(ItemListFilterAction(filteringText: ""))
 
-        guard let view = self.view,
-              let sortButtonObserver = view.sortingButtonEnabled else {
-            return
+        guard let view = self.view else { return }
+
+        let syncingObserver = self.dataStore.syncState.map {(syncState: SyncState) -> Bool in
+            return syncState == .Syncing
         }
 
+        let isManualRefreshObserver = self.itemListDisplayStore.listDisplay
```

Comment:
This is an `Observable`, not an `Observer` :p

Reply:
oops :)